DashKit/DashPumpManager.swift

```diff
@@ -454,7 +454,7 @@ public class DashPumpManager: PumpManager {
 
     private var isPumpDataStale: Bool {
         let pumpStatusAgeTolerance = TimeInterval(minutes: 6)
-        let pumpDataAge = -(state.lastStatusDate ?? .distantPast).timeIntervalSinceNow
+        let pumpDataAge = -(state.lastStatusDate ?? .distantPast).timeIntervalSince(dateGenerator())
```

Comment:
This seemed wrong to me, so I fixed it.  It was preventing me from testing stale data vs. non-stale data.